### DISCUSSION TITLE
PDF download Not working

### DISCUSSION THREAD

**Discussion by Qwili-Chandra** [afreakyelf/Pdf-Viewer#82] [General]
Looks like permission issue

**Comment by afreakyelf** [comment]
Have fixed it in new release, I will publish new build soon.

**Comment by afreakyelf** [comment]
Hi, Thank you for using this library. Above issue has been fixed in latest version.
https://github.com/afreakyelf/Pdf-Viewer/releases/tag/v2.0.1
